src/utils.js

```diff
@@ -24,7 +24,20 @@ export function psKill (options = { command: '' }) {
       if (error) {
         return reject(error)
       }
-      return resolve(result)
+
+      const pid = result && result.pid
+
+      if (!pid) {
+        return reject(new Error('Could not found target process to kill'))
```

Comment:
`Could not found target process to kill` --> `The target process does not appear to be running.`